.rspec

```diff
@@ -1,4 +1,5 @@
 -I.
 --color
---profile
---format progress
+#--profile
+#--format progress
+--format d
```

Comment:
Even though this is my preferred way to run specs locally, are we sure we want to commit this? It will make the CI logs more verbose.